src/platform/telemetry/vscode-node/microsoftExperimentationService.ts

```diff
@@ -146,6 +146,17 @@ class DevDeviceIdFilterProvider implements IExperimentationFilterProvider {
 	}
 }
 
+class Fcv1FilterProvider implements IExperimentationFilterProvider {
```

Comment:
Maybe do not create a new filter provider and instead add the filter in `GithubAccountFilterProvider`